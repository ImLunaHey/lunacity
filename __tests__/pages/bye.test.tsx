import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Bye from '../../src/pages/bye';

jest.mock('react-i18next', () => ({
  useTranslation: jest.fn(() => ({
    t: jest.fn((text: string, args) => (args ? `${text} ${JSON.stringify(args, null)}` : text)),
    ready: true,
  })),
}));

describe('Bye', () => {
  it('when unauthenticated it renders a goodbye message', () => {
    const { container } = render(<Bye />);
    expect(container).toMatchSnapshot();
  });
});
