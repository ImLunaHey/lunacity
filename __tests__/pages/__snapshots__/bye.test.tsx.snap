// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bye when unauthenticated it renders a goodbye message 1`] = `
<div>
  <div>
    <h1>
      Account Deactivated
    </h1>
    <p>
      Your account has been deactivated.
    </p>
    <p>
      In 30 days your account will be deleted. If you would like to reactivate your account before that happens, just signin again.
    </p>
  </div>
</div>
`;
